pkg/console/subresource/configmap/configmap.go

```diff
@@ -155,6 +159,17 @@ func servingInfo() yaml.MapSlice {
 	}
 }
 
+func consoleProviders(providers operatorv1.ConsoleProviders) yaml.MapSlice {
```

Comment:
This may not need testing if our main public func is well tested.  See my other test comment about adding more rows to the table test.